framework/source/class/qx/bom/webfonts/Validator.js

```diff
@@ -307,7 +306,7 @@ qx.Class.define("qx.bom.webfonts.Validator", {
       }
 
       var elem = document.createElement("span");
-      elem.innerHTML = comparisonString || qx.bom.webfonts.Validator.COMPARISON_STRING;
+      elem.innerHTML = this.getComparisonString();
```

Comment:
Line 296 still has comparisonString as a parameter, but it appears to be not used in the function. Change parameter list to be only (fontFamily) ?